dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/minmax/hibernate/HibernateMinMaxDataElementStore.java

```diff
@@ -117,4 +151,155 @@ public void delete( Collection<DataElement> dataElements, Collection<Organisatio
             setParameterList( "dataElements", dataElements ).
             setParameterList( "organisationUnits", organisationUnits ).executeUpdate();
     }
+
```

Comment:
All of these methods feel like they are a bit generic and re-inventing. Do we need it, can we keep it simpler? Or, do we generify it and move it to core?